.circleci/config.yml

```diff
@@ -93,6 +93,14 @@ build_and_deploy: &build_and_deploy
           composer install --no-suggest --no-interaction
           # TODO: should only deploy non-dev packages
           # --no-dev
+    - run:
```

Comment:
I'm approving this, but there is a caching step that I think we might need to revisit in a future story so that we only rebuild the theme when it has changed. 🤔

Reply:
That's going to be a bit trickier, as we will have to do something like an md5 hash checksum on every file in the theme directory to determine if a change happened (with the dependencies, it just checks the package-lock file). I'm wondering if its really worth it, as the build of the theme and pl takes 17s, and running a checksum on the theme directory then restoring a cache would also take a few seconds. The gain might be less than 10s.